src/extlibs/enlsvg/LineOfSightScanner.h

```diff
@@ -0,0 +1,132 @@
+#ifndef _LINE_OF_SIGHT_SCANNER_H_
```

Comment:
this file is a duplicate of `line_of_sight_scanner.h` and we prefer the line_of_sight_scanner naming